incubator/ethereum/README.md

```diff
@@ -0,0 +1,80 @@
+# ethereum
+
+[Ethereum](https://www.ethereum.org/) is a decentralized platform for building applications on the blockchain.
+
+## TL;DR;
+
+```console
+$ helm install incubator/ethereum
+```
+
+## Introduction
+
+This chart deploys a private [Ethereum](https://www.ethereum.org/) network onto a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager. For more information on running a private network, refer to [this](https://github.com/ethereum/go-ethereum/wiki/Private-network). The private Ethereum network deployed with this chart is comprised of 4 different components: 
+
+1. bootnode: used for node discovery
+2. ethstats: [Ethereum Network Stats](https://github.com/cubedro/eth-netstats)
+3. geth-miner: Geth miner nodes
+3. geth-tx: Geth transaction nodes with mining disabled whose responsbility is to respond to API (websocket, rpc) queries
+
+## Prerequisites
+
+* Kubernetes 1.8
+
+## Installing the Chart
+
+The chart can be installed as follows:
+
+```console
+$ helm install --name my-release incubator/ethereum
+```
+
+The command deploys a private Ethereum network in the default configuration. The configuration](#configuration) section lists various ways to override default configuration during deployment.
+
+> **Tip**: List all releases using `helm list`
```

Comment:
This seems a bit out of place IMO. Should be in general helm documentation.

Reply:
I was basing this `README` off of the existing charts in this repo. I can remove.

FYI, here is one: https://github.com/kubernetes/charts/blob/5e40a792c64fbd1390d3ce80dfb228949fb49161/stable/centrifugo/README.md#installing-the-chart